src/main/java/it/gov/pagopa/pu/debtpositions/repository/InstallmentNoPIIRepository.java

```diff
@@ -76,4 +76,12 @@ List<InstallmentNoPII> getByOrganizationIdAndNav(@Param("organizationId") Long o
   //Schema configuration needed to resolve the DTO generated by SpringDataRest
   @ApiResponse(responseCode = "200", content = @Content(schema = @Schema(ref = "CollectionModelInstallmentNoPII")))
   List<InstallmentNoPII> findByReceiptId(long receiptId);
+
+  @Query("select dpto.code from InstallmentNoPII i " +
+    "join PaymentOption po on i.paymentOptionId = po.paymentOptionId " +
+    "join DebtPosition dp on po.debtPositionId = dp.debtPositionId " +
+    "join DebtPositionTypeOrg dpto on dpto.debtPositionTypeOrgId = dp.debtPositionTypeOrgId " +
+    "where i.installmentId = :installmentId")
+  String getDebtPositionTypeOrgCode(@Param("installmentId") long installmentId);
```

Comment:
annotation not need if param as same name of the variable
```suggestion
  String getDebtPositionTypeOrgCodeByInstallmentId(long installmentId);
```